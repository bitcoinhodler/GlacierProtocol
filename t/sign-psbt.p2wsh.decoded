{
  "txid": "839564be4464ffd624434dc69336edf1c2fc890c6c5181d8c846015e46653496",
  "hash": "fe7120b745b4049285622de3c376f56971c59b6a8fb9853254cd9c1b8b1f4513",
  "version": 2,
  "size": 370,
  "vsize": 154,
  "weight": 616,
  "locktime": 0,
  "vin": [
    {
      "txid": "24ed6fb2ee8562b41f76f8550294a441f339ba78c5d1b8281112832c4c26f076",
      "vout": 1,
      "scriptSig": {
        "asm": "",
        "hex": ""
      },
      "txinwitness": [
        "",
        "30440220504c6544910c1fcd6c3cc1661d11284a694546282aaa345f62487657577ecfea022047874c30564a3091bf1b73b5f5ff2773071059d0d53ae5e486ef4528e71f84b301",
        "304402202e4e76aee3bedf776208675f0d9b603239104604a4408661ebd614f101fc95d902205d58a5504d037f114e7c224eb2242e88091dddba95e1748aaa64f8caaf93140201",
        "522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae"
      ],
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.04999692,
      "n": 0,
      "scriptPubKey": {
        "asm": "0 a410c764ac8da12b578a06027f11e02f07b7e1d3",
        "desc": "addr(bcrt1q5sgvwe9v3ksjk4u2qcp87y0q9urm0cwnrkhp23)#5da9chcd",
        "hex": "0014a410c764ac8da12b578a06027f11e02f07b7e1d3",
        "address": "bcrt1q5sgvwe9v3ksjk4u2qcp87y0q9urm0cwnrkhp23",
        "type": "witness_v0_keyhash"
      }
    }
  ]
}
