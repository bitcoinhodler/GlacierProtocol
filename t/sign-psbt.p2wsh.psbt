cHNidP8BAFICAAAAAZb+SoaLQGohNuRB3mkinNndIxR3FQlhfFtojZv9UqamAQAAAAD/////AQxKTAAAAAAAFgAUpBDHZKyNoStXigYCfxHgLwe34dMAAAAAAAEAlAIAAAABFTZIUpIJ+bMb5wgjworVfC0rbcg9kQviHI0ZJLsShRcAAAAAFxYAFOyJ/+6MSQvwQWmud4kPoWMsjJfh/////wLaCAYAAAAAABYAFHI+Kh0raYmP9FFp19SHvpWwHVCDQEtMAAAAAAAiACBlyGPHUDMcwD77TayiF+eSXP+H9enKmSdWQjS+DWJ46AAAAAABAStAS0wAAAAAACIAIGXIY8dQMxzAPvtNrKIX55Jc/4f16cqZJ1ZCNL4NYnjoAQWLUiED0U3c+2gX9VeWlbuz6z4YWIe/KUKwMeb3FjQ7j+fp6OIhAo/NRvhhSyy/MYCWloJCoeIrz7bY8rbck5yMJ8NHspN7IQMVrLVQEg9M3LRg1cSQgKtQjKTM2N7eysIv7qyM0BfUwSECKwY+4vIvnhmCwUD+d4ZyxoMRGnltzM29R7tl49YIqYNUrgAA
