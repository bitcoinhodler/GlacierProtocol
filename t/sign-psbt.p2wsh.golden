Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?Input a filename located in the current directory which contains the PSBT:
0.05000000 BTC in unspent inputs from cold storage address bcrt1qvhyx836sxvwvq0hmfkk2y9l8jfw0lpl4a89fjf6kgg6turtz0r5q2wgvlj
0.04999692 BTC going to destination address bcrt1q5sgvwe9v3ksjk4u2qcp87y0q9urm0cwnrkhp23
Fee amount: 0.00000308
Confirm? (y/n): 
How many private keys will you be signing this transaction with (at least 2 required to complete transaction)?
#: Key #1: Key #2: 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.05000000 BTC in unspent inputs from cold storage address bcrt1qvhyx836sxvwvq0hmfkk2y9l8jfw0lpl4a89fjf6kgg6turtz0r5q2wgvlj
0.04999692 BTC going to destination address bcrt1q5sgvwe9v3ksjk4u2qcp87y0q9urm0cwnrkhp23
Fee amount: 0.00000308

Signing with private keys: 
cQCrT9Ncs9729ao7jbmAWrD9z7tF64s2yKzmD6nkiLAi9sXVZWAn
cP65UeSDZPiTLB6CBwasWv9oJYEjRgQXhswfwcT9HscEKDcEbgy4


Confirm? (y/n): 
Calculating transaction...

Final fee rate: 2 satoshis per vbyte

Raw signed transaction (hex):
020000000001013ac0fa7023a2b6ef5a183e544d6a97a0e79913226c946e2631d5ab52356fbd9f0100000000ffffffff010c4a4c0000000000160014a410c764ac8da12b578a06027f11e02f07b7e1d304004730440220134b6b0474cbb3454d2384a1c6c0d5101dde5f9bd2960954cc108cc3773b50be0220155a2180ef2e9bbf01c54405939e71a2fd3f968aef42394018ca7fc9e9eb978f014730440220542cf913cd8e0c36f473ebaf6a0ccb0e2759d6d35d492b6e3d80b75f0852b91c02202bbd64358abb53cce0b7701d7d595764c773ee474e96bf2b87d765ff22f1865f018b522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae00000000

Transaction fingerprint (md5):
cc67f284cf85982196b02eec3319a90b
QR code for transaction written to transaction.png
